src/javascript/components/SitemapPanel.scss

```diff
@@ -33,67 +33,12 @@
     padding-bottom: 30px;
 }
 
-.sitemapIndexFileTitle {
```

Comment:
Refactored sitemap index file related css to `panelSections/SitemapIndexLink/SitemapIndexLink.scss`